internal/serverinstall/k8s.go

```diff
@@ -768,6 +775,9 @@ func (i *K8sInstaller) Uninstall(ctx context.Context, opts *InstallOpts) error {
 			LabelSelector: "app=" + serverName,
 		},
 	)
+	if err != nil {
+		return err
```

Comment:
We should display a readable error message (like you did above) here before returning the err back. Similar to the other one, it would be an error creating a statefulset watcher.